.github/workflows/test_latest_repo_change.yml

```diff
@@ -0,0 +1,63 @@
+name: Test latest changes to other repos
+
+on:
+  schedule:
+    # 05:00 PM UTC / 01:00 PM EST
+    - cron: "0 17 * * *"
+  workflow_dispatch:
+
+env:
+  RECENT: 90000 # seconds
+  WORKFLOW: unit_tests_with_latest_deps.yml
+  GH_TOKEN: ${{ secrets.REPO_SCOPED_TOKEN }}
```

Comment:
is this token used?

Reply:
yes, the gh cli uses this token to make authorized requests

Reply:
`REPO_SCOPED_TOKEN` has the permissions necessary to trigger other workflows as opposed to `GITHUB_TOKEN`